### PR TITLE
Added Infix to Postfix Convertor using stack cpp

### PR DESCRIPTION
**Pull request by chiragchandnani10**

### CODE REVIEW

**Review by ritikbanger — CHANGES_REQUESTED** [review]

File: 30. CPP Programs/Infix_To_Postfix.cpp [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+                    
+                
+
+            }
+           
+
+
+            }
+            // while(!st.empty()&&(precedence(st.top())>precedence(s[i]))){
+            //     res+=st.top();
```
[the remark this answers is not in the document]
Comment (chiragchandnani10) [reply 30. CPP Programs/Infix_To_Postfix.cpp]: > Please remove the commented code if it is not useful

Done @ritikbanger Please mark it hacktoberfest-accepted! Thanks a lot!

[the remark this answers is not in the document]
Comment (chiragchandnani10) [reply 30. CPP Programs/Infix_To_Postfix.cpp]: @ritikbanger I have ran the code in my PC and its working now. Could you please merge this PR and mark it hacktoberfest-accepted? Thank you so much and god bless you.

**Comment by chiragchandnani10** [comment]
@ritikbanger I have ran the code in my PC and its working now. Could you please merge this PR and mark it hacktoberfest-accepted? Thank you so much and god bless you. (I am sorry to write so many times, this is my first open source experience)

**Comment by ritikbanger** [comment]
Don't forget to star this repository and share on your social media about this repo for hacktoberfest. Thank you for the contribution.

**Pull request merged**